apps/frontend/app/components/flows/DefaultExperiments/ExpJSONs/DefaultExpJSONs.tsx

```diff
@@ -0,0 +1,78 @@
+
+export const addNumsExpData = {
```

Comment:
Probably a lot of fields that could be removed to make this cleaner. Finished, expId, etc.